app/controllers/items_controller.rb

```diff
@@ -17,13 +18,18 @@ def show
   end
 
   def new
-    @item = Item.new(status: 'Available')
+    @item_type = ItemType.find ( params[:item_type_id])
+    @item = Item.new(
+            status: 'Available',
+            item_type_id: params[:item_type_id])
+
     @page_title = "New Item"
```

Comment:
I don't really understand this. The controller shouldn't really be concerned with non-dynamic presentation stuff like this. I'd just hardcode it in the view like all the other text things until you've got translations, and then i'd put the call to the translation in the view too, not the controller.